cfme/v2v/infrastructure_mapping.py

```diff
@@ -105,6 +105,7 @@ class general(InfraMappingCommonButtons):  # noqa
         plan_type = BootstrapSelect("targetProvider")
         name_help_text = Text(locator='.//div[contains(@id,"name")]/span')
         description_help_text = Text(locator='.//div[contains(@id,"description")]/span')
+        alert = Text('.//div[contains(@class, "alert")]')
```

Comment:
**Question** Why not use the `FlashMessage` class in widgetastic_patternfly? So all the methods would be available to you? Some updates would be needed to support v2v pages

Reply:
You could maybe inherit from that class in `wigetastic_manageiq` and overwrite the `messages` property since it looks like the locator here is a bit different. 

https://github.com/RedHatQE/widgetastic.patternfly/blob/master/src/widgetastic_patternfly/__init__.py#L252

Reply:
This alert is not same as flash message . hence flash messages cannot be used .
![Screenshot from 2019-09-13 11-29-43](https://user-images.githubusercontent.com/3866976/64885785-d0f8bf00-d619-11e9-8e62-d8743cf6954c.png)

Reply:
![Screenshot from 2019-09-13 11-30-54](https://user-images.githubusercontent.com/3866976/64885851-fab1e600-d619-11e9-9dc0-742b82100705.png)